torch/_inductor/lowering.py

```diff
@@ -343,11 +343,9 @@ def wrapped(*args, **kwargs):
             unpacked = True
             args = args[0]
 
-        # kwargs tensors not supported yet unless it's a fallback op
         if not all(
             (fn in fallbacks or in_namespace(fn, "_c10d_functional")) for fn in aten_fn
         ):
-            assert not any(isinstance(x, TensorBox) for x in kwargs.values())
```

Comment:
Sorry, maybe my comment was misleading. Everything should "mostly work" but it still requires a couple changes. Clearly we do not have any kwarg tensors lowerings right now otherwise they would have failed prior to this pr.

I think we also want to pass the kwargs to `transform_args` so they participate in type promotion/broadcasting.

Reply:
Done, hopefully! Please take another look as you have time.